src/components/list_resources_panel.tsx

```diff
@@ -40,125 +63,154 @@ export class ListResourcesPanel extends React.Component<Props, State> {
+                        <MenuItem style={styles.selectItem} value={ResourceType.Model}>Models</MenuItem>
                     </Select>
+                    <Box flexGrow={1}></Box>
+                    <IconButton disabled={this.state.isLoading} size="small">
+                        <Icon>add</Icon>
+                    </IconButton>
+                    <IconButton disabled={this.state.isLoading} size="small" onClick={(_) => { this.refresh() }}>
+                        <Icon>refresh</Icon>
+                    </IconButton>
                 </Toolbar>
 
-
                 {(this.state.resourceType == ResourceType.Dataset) ? (
                     <ListResourcesTable
                         columns={[
+                            {
+                                field: 'datasetType',
+                                title: '',
+                                render: rowData => this.iconForDatasetType(rowData.datasetType),
+                                fixedWidth: 30,
+                                sorting: false
+                            },
                             {
                                 field: 'displayName',
-                                title: 'Name',
-                                render: rowData => (<ListItem dense style={{ padding: 0 }}>
-                                    <TableChartOutlinedIcon></TableChartOutlinedIcon>
-                                    <span style={{ textOverflow: 'ellipsis' }}> {'\u00A0\u00A0' + rowData.displayName}</span>
-                                </ListItem>),
+                                title: 'Name'
                             },
                             {
-                                title: 'Examples', field: 'exampleCount', type: ColumnType.Numeric,
-                                minShowWidth: 380
+                                title: 'Rows', field: 'exampleCount', type: ColumnType.Numeric,
+                                minShowWidth: breakpoints[1],
+                                fixedWidth: 80
                             },
                             {
                                 title: 'Created at', field: 'createTime', type: ColumnType.DateTime,
                                 rightAlign: true,
-                                minShowWidth: 250
+                                minShowWidth: breakpoints[0]
                             }
                         ]}
-                        data={datasets}
+                        data={this.state.datasets}
                         onRowClick={(rowData) => { this.props.context.manager.launchWidgetForId(rowData.id, rowData) }}
                         isLoading={this.state.isLoading}
-                        height={this.props.height - 100}
+                        height={this.props.height - 88}
                         width={this.props.width}
                     />
                 ) : (
                         <ListResourcesTable
                             columns={[
```

Comment:
It could be nice to have an icon here for models as well to keep it consistent with the datasets list? just a suggestion. definately don't have to..